Project.toml

```diff
@@ -19,6 +19,7 @@ FastClosures = "9aa1b823-49e4-5ca5-8b0f-3971ec8bab6a"
 FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 FunctionWrappersWrappers = "77dc65aa-8811-40c2-897b-53d922fa7daf"
+LineSearches = "d3d80556-e9d4-5f37-9878-2ab0fcc64255"
```

Comment:
Compat

Reply:
Should we keep the dependency then?

Reply:
@YingboMa bump

Reply:
yes, keep the dependency just put a compat on it.

Reply:
I added it.